Jenkinsfile

```diff
@@ -58,12 +59,14 @@ elifePipeline {
             }
 
             elifeMainlineOnly {
-                stage 'Push image', {
+                stage 'Push images', {
+                    assetsImage.push()
                     image.push()
                 }
 
                 stage 'Approval', {
                     elifeGitMoveToBranch commit, 'approved'
+                    assetsImage.tag('approved').push()
```

Comment:
Bit redundant.

Reply:
Would drop the `approved` tag for both images?